### PR TITLE
Repository Get Archive Link

### LINKED ISSUE #748: Get Archive Link for Repository Contents
Reference: https://developer.github.com/v3/repos/contents/#get-archive-link

This should belong under `IRepositoryContentsClient` but the workflow for turning a `302` into actual content might need a bit of thought.

### PR DESCRIPTION
**Pull request by alfhenrik**
Partial fix for #748 

- [x] Get link to the archive
- ~~[ ] Get the **actual** content of the archive.... :santa: :gift:~~ Will send a separate PR for this one

### CODE REVIEW

**Comment by NikolayIT** [comment]
When this will be merged?

File: Octokit.Reactive/Clients/ObservableRepositoryContentsClient.cs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using System.Reactive.Threading.Tasks;
+using Microsoft.SqlServer.Server;
```
Comment (shiftkey) [inline Octokit.Reactive/Clients/ObservableRepositoryContentsClient.cs]: Wait, what?

**Comment by shiftkey** [comment]
@NikolayIT not sure - were you waiting the complete "get actual content" step as well? that code hasn't been implemented yet...

**Comment by alfhenrik** [comment]
That'd be my fault, I was trying to come up w with a good solution for getting the contents of the file and then we got super busy at work.

I might have some time this weekend to have another look at it again.
________________________________

File: Octokit.Reactive/Clients/ObservableRepositoryContentsClient.cs (hunk 1 above)
Comment (alfhenrik) [inline Octokit.Reactive/Clients/ObservableRepositoryContentsClient.cs]: Uhm.......😲

I have no idea where that would have come from...I will of course remove said reference quicker than quick 😁
________________________________

**Comment by NikolayIT** [comment]
@shiftkey I only need link to the archive. :)

**Comment by shiftkey** [comment]
@alfhenrik if you find yourself still struggling to make time for it, I'm happy to short this PR and just merge in the archive link stuff.

**Comment by alfhenrik** [comment]
Happy for you to merge this PR now, and I'll send a new one for the content stuff :+1:

File: Octokit.Tests.Integration/Clients/RepositoryContentsClientTests.cs [hunk 2]
```diff
@@ -128,5 +128,53 @@ public class TheGetContentsMethod
                 Helper.DeleteRepo(repository);
             }
         }
+
+        [IntegrationTest]
+        public async Task GetsArchiveLinkAsTarball()
+        {
+            var github = new GitHubClient(new ProductHeaderValue("OctokitTests"))
```
Comment (shiftkey) [inline Octokit.Tests.Integration/Clients/RepositoryContentsClientTests.cs]: :lipstick: just use `Helper.GetAuthenticatedClient()` for these integration tests

Comment (alfhenrik) [inline Octokit.Tests.Integration/Clients/RepositoryContentsClientTests.cs]: That's what `ctrl-c, ctrl-v` gets you, shouldn't trust previous methods in same class :innocent: (_just kidding_)

**Comment by shiftkey** [comment]
@NikolayIT this just went out in `v0.11` - https://www.nuget.org/packages/Octokit/0.11.0

**Comment by NikolayIT** [comment]
@shiftkey, awesome! Thanks!

**Pull request merged**